### PR TITLE
improve color management and design

### PR DESCRIPTION
**Pull request by noahschoenthal**

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -1,15 +1,18 @@
 :root {
   --primary: #d93434;
   --primary-tint: #dd4848;
-  --primary-shade: rgb(127, 51, 51);
+  --primary-carouselarrow: rgb(119, 35, 35);
+  --primary-gradebooklet: rgb(127, 51, 51);
+  --primary-features: rgb(217, 52, 52, 1);
+  --primary-comingsoon: rgb(127, 51, 51);
```
Comment (luca-bruegger) [inline src/app.css]: That's a no-no. Primary variables are here to define things once; Please remove the duplicate assignments like `--primary-coming-soon` and `--primary-gradebooklet`. For further clarification, call the variable: `--secondary-content: rgb(127, 51, 51);` then `--primary-content: rgb(119, 35, 35);` as well as the third one `--tertiary-content: rgb(217, 52, 52, 1);`

File: src/app.css [hunk 2]
```diff
@@ -20,7 +23,18 @@ body {
...
+  background-color: var(--primary-gradebooklet);
+}
+
+#home-features {
+  background-color: var(--primary-features);
+}
+
+#home-comingsoon {
+  background-color: var(--primary-comingsoon);
 }
```
Comment (luca-bruegger) [inline src/app.css]: Please do not work with ids in this specific case. Fix it like this (and keep in mind that you have to do this for `secondary` and `tertiary` colors aswell):
`
.primary-content {
   background-color: var(--primary-content);
}
`

File: src/app.css [hunk 3]
```diff
@@ -278,7 +291,7 @@ a {
...
 
 .arrows path {
-  stroke: var(--primary);
+  stroke: rgb(74, 59, 59, 1);
   fill: transparent;
```
Comment (luca-bruegger) [inline src/app.css]: No-no again. Define a variable and use it over here.

File: src/app.html [hunk 4]
```diff
@@ -106,7 +106,7 @@ <h2>Contact</h2>
 
   <div class="content swiper" keydown.delegate="handleKeyInput($event)">
     <div class="swiper-wrapper">
-      <div id="home" class="swiper-slide">
+      <div id="home-gradebooklet" class="swiper-slide">
```
Comment (luca-bruegger) [inline src/app.html]: My bad that i've used id's. But since that's long ago I learnt more and now I recommend to use the classes, we defined above.

**Review by luca-bruegger — CHANGES_REQUESTED** [review]
Looks good. Some things left. DO NOT MERGE THE package-lock.json.

File: src/app.html (hunk 4 above)
Comment (noahschoenthal) [reply src/app.html]: @luca-bruegger yeah solved it using class look in new pushed comment.

**Review by luca-bruegger — APPROVED** [review]

**Pull request merged**